source/Calamari.Azure.CloudServices/Commands/DeployAzureCloudServiceCommand.cs

```diff
@@ -74,7 +74,9 @@ public override int Execute(string[] commandLineArguments)
             var configurationTransformer = ConfigurationTransformer.FromVariables(variables);
             var transformFileLocator = new TransformFileLocator(fileSystem);
             var replacer = new ConfigurationVariablesReplacer(variables.GetFlag(SpecialVariables.Package.IgnoreVariableReplacementErrors));
-            var jsonVariablesReplacer = new JsonConfigurationVariableReplacer();
+            var jsonVariablesReplacer = new StructuredConfigVariableReplacer(
```

Comment:
There will be changes to DI in a near-future PR. This is ugly but our DI will all be replaced soon.